### ISSUE TITLE
Feat: Need of a payment gateway for people to buy the books

### ISSUE THREAD

**Issue by kamalbuilds** [legendarykamal/Lets-go-shop#14]
This will be a mazor upgrade for the project and will be an enhancement in the functionality.

I have razorpay gateway to be implemented for the site for now.

**Comment by kamalbuilds** [comment]
This is an area open for discussion and collaberation.

**Comment by tamalCodes** [comment]
Hi there, this is a do-able thing , but for that we would need secrets like : 

```
 key_id: process.env.KEY_ID,
 key_secret: process.env.KEY_SECRET,

```

These can be a bit dangerous if shared in public !

**Comment by kamalbuilds** [comment]
yes @IAmTamal thats why we shall store it in the form of secrets so that Its not visible to everyone.

**Comment by kamalbuilds** [comment]
@IAmTamal are you interested in working in this I am creating sub issues for this .
